File: app/src/main/java/com/example/olesya/boardgames/models/ImageHolder.java
package com.example.olesya.boardgames.models;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import android.support.annotation.NonNull;

@Entity
public class ImageHolder {

    @PrimaryKey
    @NonNull
    private String imageUrl;

    public ImageHolder(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/java/com/example/olesya/boardgames/Entity/ImageHolder.kt
package com.example.olesya.boardgames.Entity

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
data class ImageHolder(var imageUrl: String) {

    @ColumnInfo(name = "id")
    @PrimaryKey(autoGenerate = true)
    var id: Long = 0

}